testing_llm/README.md

```
@@ -1,301 +1,76 @@
-Execute tests in sequence for comprehensive validation:
-
-```bash
-# Run all tests in order
-export OPENAI_BASE_URL=http://localhost:10000
-for test_file in 01_basic_proxy_test.md 02_hook_integration.md 03_streaming_test.md 04_error_handling.md 05_performance_test.md 06_end_to_end.md; do
-    echo "Executing $test_file..."
-    codex exec --yolo "Follow all instructions in $test_file and report results"
-    echo "Completed $test_file"
-done
+/tmp/codex_plus/cereb_conversion/test_evidence/<test_name>/
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)